### ISSUE TITLE
[Failing test] Roslyn.VisualStudio.IntegrationTests.Basic.BasicReferenceHighlighting.Highlighting

### ISSUE THREAD

**Issue by 333fred** [dotnet/roslyn#18705]
https://ci.dot.net/job/dotnet_roslyn/view/Official%20Builds/job/master/job/windows_release_vs-integration/760/
https://ci.dot.net/job/dotnet_roslyn/view/Official%20Builds/job/master/job/windows_debug_vs-integration/768/

```
    Roslyn.VisualStudio.IntegrationTests.Basic.BasicReferenceHighlighting.Highlighting
      
      Expected:
      [77..80)
      Actual:
      [77..80)
      [91..94)
      Differences:
          [77..80)
      ++> [91..94)
      
      Expected: True
      Actual:   False
      Stack Trace:
           at Roslyn.Test.Utilities.AssertEx.SetEqual[T](IEnumerable`1 expected, IEnumerable`1 actual, IEqualityComparer`1 comparer, String message, String itemSeparator)
        q:\roslyn\src\VisualStudio\IntegrationTest\IntegrationTests\VisualBasic\BasicReferenceHighlighting.cs(54,0): at Roslyn.VisualStudio.IntegrationTests.Basic.BasicReferenceHighlighting.Verify(String marker, IDictionary`2 spans)
        q:\roslyn\src\VisualStudio\IntegrationTest\IntegrationTests\VisualBasic\BasicReferenceHighlighting.cs(39,0): at Roslyn.VisualStudio.IntegrationTests.Basic.BasicReferenceHighlighting.Highlighting()
```

![image](https://cloud.githubusercontent.com/assets/2371880/25050929/edd8593c-20fe-11e7-8377-1a91ced0b324.png)

**Comment by 333fred** [comment]
The C# version is also failing, probably related. If it's not, @rchande, please open a new issue.
```
    Roslyn.VisualStudio.IntegrationTests.CSharp.CSharpReferenceHighlighting.WrittenReference
      
      Expected:
      [59..60)
      Actual:
      
      Differences:
      --> [59..60)
      
      Expected: True
      Actual:   False
      Stack Trace:
           at Roslyn.Test.Utilities.AssertEx.SetEqual[T](IEnumerable`1 expected, IEnumerable`1 actual, IEqualityComparer`1 comparer, String message, String itemSeparator)
        q:\roslyn\src\VisualStudio\IntegrationTest\IntegrationTests\CSharp\CSharpReferenceHighlighting.cs(102,0): at Roslyn.VisualStudio.IntegrationTests.CSharp.CSharpReferenceHighlighting.Verify(String marker, IDictionary`2 spans)
        q:\roslyn\src\VisualStudio\IntegrationTest\IntegrationTests\CSharp\CSharpReferenceHighlighting.cs(59,0): at Roslyn.VisualStudio.IntegrationTests.CSharp.CSharpReferenceHighlighting.WrittenReference()
```
![image](https://cloud.githubusercontent.com/assets/2371880/25051041/88712424-20ff-11e7-9dfa-0e3e7ada1bb3.png)

**Comment by MattGertz** [comment]
Mondo escrow triage: keeping until we understand if functionality is affected.

**Comment by Pilchie** [comment]
This looks like a legitimate failure where we should update what the test expects? Related to https://github.com/dotnet/roslyn/pull/17682 maybe?

**Comment by CyrusNajmabadi** [comment]
@333fred do we have line numbers?

**Comment by CyrusNajmabadi** [comment]
i'm going to update the test to give more informatin.

**Comment by 333fred** [comment]
@CyrusNajmabadi not sure, I copied whatever information I could find in the test logs.

**Comment by CyrusNajmabadi** [comment]
Have a PR to collect more data: https://github.com/dotnet/roslyn/pull/20580

**Comment by Pilchie** [comment]
Closing until we get new failures after #20580
